test/extended/util/disruption/frontends/frontends.go

```diff
@@ -117,7 +117,7 @@ func (t *AvailableTest) Test(f *framework.Framework, done <-chan struct{}, upgra
 
 	toleratedDisruption := 0.20
 	switch {
-	case framework.ProviderIs("azure"), framework.ProviderIs("aws"), framework.ProviderIs("gce"):
+	case framework.ProviderIs("azure"), framework.ProviderIs("aws"):
```

Comment:
Needs a commented out line and a link to a bug

Reply:
Fixed